src/messages/types.rs

```diff
@@ -425,12 +425,22 @@ impl ::protobuf::reflect::ProtobufValue for KI64Pair {
 }
 
 static file_descriptor_proto_data: &'static [u8] = b"\
-    \n9github.com/tendermint/tendermint/libs/common/common.proto\x12\x06comm\
-    on\",\n\x06KVPair\x12\x0f\n\x03key\x18\x01\x20\x01(\x0cB\x02\x18\0\x12\
+    \n8github.com/tendermint/tendermint/libs/common/types.proto\x12\x06commo\
+    n\",\n\x06KVPair\x12\x0f\n\x03key\x18\x01\x20\x01(\x0cB\x02\x18\0\x12\
     \x11\n\x05value\x18\x02\x20\x01(\x0cB\x02\x18\0\".\n\x08KI64Pair\x12\x0f\
     \n\x03key\x18\x01\x20\x01(\x0cB\x02\x18\0\x12\x11\n\x05value\x18\x02\x20\
-    \x01(\x03B\x02\x18\0B\x1c\xc8\xe2\x1e\x01\xb8\xe2\x1e\x01\xf8\xe1\x1e\
-    \x01\xd0\xe2\x1e\x01\xc0\xe3\x1e\x01\xe0\xe2\x1e\x01\xa8\xe2\x1e\x01b\
+<<<<<<< HEAD:src/messages/types.rs
+<<<<<<< HEAD
+    \x01(\x03B\x02\x18\0B\x1c\xf8\xe1\x1e\x01\xe0\xe2\x1e\x01\xc8\xe2\x1e\
+    \x01\xb8\xe2\x1e\x01\xc0\xe3\x1e\x01\xd0\xe2\x1e\x01\xa8\xe2\x1e\x01b\
+=======
+    \x01(\x03B\x02\x18\0B\x1c\xd0\xe2\x1e\x01\xc8\xe2\x1e\x01\xa8\xe2\x1e\
+    \x01\xc0\xe3\x1e\x01\xb8\xe2\x1e\x01\xe0\xe2\x1e\x01\xf8\xe1\x1e\x01b\
+>>>>>>> proper From for abcistream
+=======
+    \x01(\x03B\x02\x18\0B\x1c\xd0\xe2\x1e\x01\xc8\xe2\x1e\x01\xa8\xe2\x1e\
+    \x01\xc0\xe3\x1e\x01\xb8\xe2\x1e\x01\xe0\xe2\x1e\x01\xf8\xe1\x1e\x01b\
+>>>>>>> d8183843a6a11d8ecd80a1825dc3b6a31ea48b42:src/messages/types.rs
```

Comment:
happens to the best of us ;)

Reply:
@zramsay Oops...  Because the build.rs runs every time, I didn't push the messages after the last rebase step.   Anyway, updated all messages just to be safe.  But this is a good example of needing more tests for the protobuf stuff

Reply:
#36